src/pages/Home/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Layout from '../../components/Layout/Layout';
+// eslint-disable-next-line import/extensions
+import CarsList from '../../components/CarsList/CarsList';
+
+const Home = () => (
```

Comment:
- [ ] OPTIONAL:  I couldn't find a way to logout.  Kindly add a logout button so more users can be tested.